src/test/java/uk/gov/hmcts/reform/translate/helper/DictionaryMapperTest.java

```diff
@@ -49,8 +66,8 @@ void shouldMaModelToEntityWithoutTranslationPhrase() {
         val currentPhrase = getCurrentPhrase();
         val dictionaryEntity = dictionaryMapper.modelToEntityWithoutTranslationPhrase(currentPhrase);
         assertEquals(ENGLISH, dictionaryEntity.getEnglishPhrase());
-        assertEquals(null, dictionaryEntity.getTranslationPhrase());
-        assertTrue(dictionaryEntity.getTranslationUpload() == null);
+        assertNull(dictionaryEntity.getTranslationPhrase());
```

Comment:
typo in method name - missing `p` from `Map`

`shouldMaModelToEntityWithoutTranslationPhrase()`

Reply:
Resolved in 6f43c1e51f16676d52a225ac71a4bf8f0ed05f62.